cypress/features/regression/experimental/search.feature

```diff
@@ -0,0 +1,71 @@
+Feature: Search component
```

Comment:
1. Please add TC which check the color of search icon when input is focused
2. Please add TC which check whether the input is empty after clicking on the cross icon

Reply:
added